src/main/resources/finalCopy/final_workflow_log_dir.test

```diff
@@ -0,0 +1,11 @@
+# Tests that the runtime option to preserve logs from the workflow is being honored.
+
+name: workflow_log_dir
+testFormat: workflowsuccess
+backends: [Local]
+tags: [localdockertest]
```

Comment:
I don't know what these settings mean so I didn't change them.  I'll delete these and see if it still passes